src/main/java/org/spongepowered/common/mixin/core/block/MixinBlock.java

```diff
@@ -179,4 +180,12 @@ private static void onSpawnAsEntity(net.minecraft.world.World worldIn, BlockPos
             return;
         }
     }
+
+    @Override
+    public String toString() {
+        return SpongeCatalogType.toStringHelperTranslatable(this)
```

Comment:
The method isn't called often, so I don't see a reason to lazy load it.

> If you're going to change what Block.toString() returns

I don't change it, there is no default implementaion in vanilla.